base/system/userinit/userinit.h

```diff
@@ -58,6 +58,7 @@ typedef struct
     PAGESTATE NextPage;
     RUN Run;
     IMGINFO ImageInfo;
+    struct _LIVECD_UNATTEND* Unattend;
```

Comment:
Why using a pointer instead of just an embedded structure as member?

Reply:
Didnt want to dump too much crap in userinit.h